src/components/InfoPanel.component.js

```diff
@@ -1,8 +1,42 @@
 import React, { Component } from 'react'
+import { Image } from 'react-bootstrap'
+import './../styles/App.css'
 
 class InfoPanel extends Component {
+  constructor(props) {
```

Comment:
This constructor isn't needed since the default constructor will do the same thing.